### PR TITLE
Release  0.6.5

### PR DESCRIPTION
**Pull request by benjaminpkane**

### CODE REVIEW

**Review by brimoor — APPROVED** [review]
LGTM for when the release is ready (after #686 is merged, at least).

Someone else may want to do a quick once-over on this since I contributed some docs enhancements here.

**Comment by benjaminpkane** [comment]
Yes, I will do another pass. Check docs build, etc.

**Comment by brimoor** [comment]
@benjaminpkane FYI I decided to merge https://github.com/voxel51/fiftyone/pull/686 into here instead so all release changes are in one place. I'm cool with merging this into `develop` as soon as the current unit tests pass

(It was a bit annoying to develop on a branch with a data model change that requires a DB migration w/o the corresponding version changes to the package version in `setup.py`, etc. Maybe there's an easy way to run migrations manually but this was easier for me, for now)

**Pull request merged**
